NGitLab/IProjectBadgeClient.cs

```diff
@@ -26,24 +26,32 @@ public interface IProjectVariableClient
 {
     IEnumerable<Variable> All { get; }
 
-    Variable this[string key] { get; }
+    Variable this[string key, string environmentScope = null] { get; }
 
     Variable Create(VariableCreate model);
 
     Variable Update(string key, VariableUpdate model);
 
+    Variable Update(string key, string environmentScope, VariableUpdate model);
+
     void Delete(string key);
+
+    void Delete(string key, string environmentScope);
 }
 
 public interface IGroupVariableClient
 {
     IEnumerable<Variable> All { get; }
 
-    Variable this[string key] { get; }
+    Variable this[string key, string environmentScope = null] { get; }
```

Comment:
```suggestion
```suggestion
    Variable this[string key] { get; }
    
    Variable this[string key, string environmentScope] { get; }
```
```

Reply:
Addressed by 824f7801eb4603bed79d45ba78a85e11d4212514